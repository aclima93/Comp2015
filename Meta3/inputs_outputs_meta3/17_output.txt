Line 7, col 12: Symbol A not defined
