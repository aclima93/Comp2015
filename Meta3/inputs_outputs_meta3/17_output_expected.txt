===== Outer Symbol Table =====
boolean	_type_	constant	_boolean_
integer	_type_	constant	_integer_
real	_type_	constant	_real_
false	_boolean_	constant	_false_
true	_boolean_	constant	_true_
paramcount	_function_
program	_program_

===== Function Symbol Table =====
paramcount	_integer_	return

===== Program Symbol Table =====
x	_integer_
y	_integer_
readexpression	_function_
readoperand	_function_
gcd	_function_

===== Function Symbol Table =====
readexpression	_integer_	return
this	_real_
op	_real_

===== Function Symbol Table =====
readoperand	_integer_	return

===== Function Symbol Table =====
gcd	_real_	return
a	_integer_	param
b	_integer_	param